internal/serve/httphandler/forgot_password_handler.go

```diff
@@ -74,24 +102,14 @@ func (h ForgotPasswordHandler) ServeHTTP(w http.ResponseWriter, r *http.Request)
 		}
 	}
 
-	// validate request
-	v := validators.NewValidator()
-
-	v.Check(forgotPasswordRequest.Email != "", "email", "email is required")
-
-	if v.HasErrors() {
-		httperror.BadRequest("request invalid", err, v.Errors).Render(w)
-		return
-	}
-
 	resetToken, err := h.AuthManager.ForgotPassword(ctx, forgotPasswordRequest.Email)
-	// if we don't find the user by email, we just return an ok response
+	// If we don't find the user by email, we just return an ok response
 	// to prevent malicious client from searching accounts in the system
 	if err != nil {
 		if errors.Is(err, auth.ErrUserNotFound) {
-			log.Ctx(ctx).Errorf("error in forgot password handler, email not found: %s", forgotPasswordRequest.Email)
+			log.Ctx(ctx).Errorf("Email not found in forgot password handler: %s", forgotPasswordRequest.Email)
 		} else if errors.Is(err, auth.ErrUserHasValidToken) {
-			log.Ctx(ctx).Errorf("error in forgot password handler, user has a valid token")
+			log.Ctx(ctx).Errorf("User has a valid token in forgot password handler")
```

Comment:
🤔 It's not clear what that means. After looking the code, it seems the forgot password will generate a maximum of one token per 20 minutes. meaning the user would need to wait for 20min before they can generate a new token.

💡  Can you improve the log message to explain that, so an admin can debug the problem without knowledge of the codebase?